conf/app.routes

```diff
@@ -43,6 +43,9 @@ GET         /agents/optout/choose-taxyear                                 contro
 POST        /optout/choose-taxyear                                        controllers.optOut.OptOutChooseTaxYearController.submit(isAgent: Boolean = false)
 POST        /agents/optout/choose-taxyear                                 controllers.optOut.OptOutChooseTaxYearController.submit(isAgent: Boolean = true)
 
+GET         /optout/cancelled                                             controllers.optOut.OptOutCancelledController.show()
+
```

Comment:
My thinking and guess is we can stick with one since the new way of implementing auth and determining the type of user would provide the flag for determining if the agent is a user or not. 

We then can control the content for users or agents using this flag. We could render different content or entirely new views using 
`user.isAgent()`
I am guessing

Maybe we need another route to be consistent with the rest of the service but it would just be only for the url to have /agent/ in it.

I'll ask around and get back to you.

Reply:
adding and splitting out the routes
https://hmrcdigital.slack.com/archives/C050V2P86BC/p1731924705093209